test/unit/pool/test-destroy-connection-timeout.js

```diff
@@ -0,0 +1,25 @@
+
+var assert     = require('assert');
+var common     = require('../../common');
+var Connection = common.Connection;
+var pool       = common.createPool({port: common.fakeServerPort});
+
+var server = common.createFakeServer();
+
+server.listen(common.fakeServerPort, function (err) {
+  assert.ifError(err);
+  pool.getConnection(function (err, connection) {
+    assert.ifError(err);
+    process.nextTick(function() {
+      connection.destroy();
+      pool.end(function (err) {
+        assert.ifError(err);
+        server.destroy();
+      })
+    })
+  })
+});
+
+setTimeout(function () {
+  throw new Error("PoolConnection.destroy left timers active after teardown")
+}, 5000).unref()
```

Comment:
The `.unref` method does not exist in 0.6 and 0.8, which are two fully supported versions of Node.js for the current major version of this module. Can you achieve the goal of this test without the unref?